### PR TITLE
Issue #42 DocetUtils: Error on Java 9

### PR DESCRIPTION
**Pull request by mcRoot**
This should be a fix to the the issue, please @eolivelli @nicoloboschi review it.

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -63,6 +63,7 @@
         <libs.junit>4.12</libs.junit>
         <libs.servletapi>3.1.0</libs.servletapi>
         <libs.flyingsaucer.pdf>9.1.7</libs.flyingsaucer.pdf>
+        <libs.netty4>4.1.12.Final</libs.netty4>
```
Comment (eolivelli) [inline pom.xml]: let's use the latest and gratest Netty version

File: .gitignore [hunk 2]
```diff
@@ -3,3 +3,4 @@
 **/.settings/
 **/.classpath
 **/target/
+**/bin
```
Comment (eolivelli) [inline .gitignore]: is this change necessary ?

File: docet-core/pom.xml [hunk 3]
```diff
@@ -93,5 +93,11 @@
                 </exclusion>
             </exclusions>
         </dependency>
+        <dependency>
+            <!-- usiamo solo netty buffer -->
+            <groupId>io.netty</groupId>
+            <artifactId>netty-all</artifactId>
```
Comment (eolivelli) [inline docet-core/pom.xml]: I think we only need netty-buffer for this project, on downstream project which use netty-all it won't be a problem, but for Docet it is a too big dependency

File: .gitignore (hunk 2 above)
Comment (mcRoot) [reply .gitignore]: building with mvn on jdd-9 container saw a few bin folders coming out from the mist: however see no reason not to add this exclusion, no harm.

File: pom.xml (hunk 1 above)
Comment (mcRoot) [reply pom.xml]: ok

File: docet-core/pom.xml (hunk 3 above)
Comment (mcRoot) [reply docet-core/pom.xml]: good, makes sense.

**Comment by mcRoot** [comment]
@eolivelli  changes ready for being reviewed.
